aws/s3.tf

```diff
@@ -123,3 +123,94 @@ module "s3_bucket" {
 
   tags = local.tags
 }
+# aws S3 backup //////////////////////
+resource "aws_backup_vault" "datagrok_public_vault" {
+  name = var.s3_backup_vault_name
```

Comment:
We already have a lot of variables; why would we need another one?
Use naming prefixes instead, like for logging bucket https://github.com/datagrok-ai/tf-module-datagrok-core/pull/26/files#diff-bfa811139a4cedd8c743561172e24c14f800b2fd054b137f984598b3d4195706R10